api/pacman-api-admin/src/main/java/com/tmobile/pacman/api/admin/repository/service/plugins/AbstractPluginService.java

```diff
@@ -34,20 +35,31 @@
 import com.tmobile.pacman.api.admin.domain.PluginParameters;
 import com.tmobile.pacman.api.admin.domain.PluginResponse;
 import com.tmobile.pacman.api.admin.exceptions.PacManException;
+import com.tmobile.pacman.api.admin.exceptions.PluginApiResponseException;
 import com.tmobile.pacman.api.admin.repository.AccountsRepository;
 import com.tmobile.pacman.api.admin.repository.model.AccountDetails;
 import com.tmobile.pacman.api.admin.repository.model.ConfigProperty;
+import com.tmobile.pacman.api.admin.repository.service.AssetGroupService;
+import com.tmobile.pacman.api.admin.repository.service.ConfigPropertyService;
+import com.tmobile.pacman.api.admin.repository.service.DataCollectorSQSService;
+import com.tmobile.pacman.api.admin.repository.service.UserPreferencesService;
 import com.tmobile.pacman.api.admin.util.AdminUtils;
 import com.tmobile.pacman.api.commons.config.CredentialProvider;
 import com.tmobile.pacman.api.commons.repo.ElasticSearchRepository;
 import org.apache.commons.lang.StringUtils;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpUriRequest;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
```

Comment:
Do you really need all these new imports?

Reply:
I moved All new plugin service to a plugin package in service